### DISCUSSION TITLE
I cannot access your site from Uganda

### DISCUSSION THREAD

**Discussion by DaliDalmas** [apache/airflow#22180] [General]
### Describe the issue with documentation

https://airflow.apache.org/

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Reply by potiuk** [reply]
I am afraid there is nothing we can do about it. Use VPN if your country blocks access.
